### ISSUE TITLE
Spacing between list items

### ISSUE THREAD

**Issue by 3e9b** [kepano/obsidian-minimal#126]
I really would like to have a an option in Minimal Theme Settings to change spacing between list items in Live Preview (and Reading View if possible). Visually it just looks less cluttered when you can control item spacing separately from the line height.

Thanks!

**Comment by kepano** [comment]
For now you can use a snippet like this:

```
.markdown-preview ul>li,
.mod-cm6 .HyperMD-list-line.cm-line {
  padding-top:0.25em;
  padding-bottom:0.25em;
}
```

**Comment by kepano** [comment]
This is now available as of 4.4.5
